build/parser.mjs

```diff
@@ -941,11 +941,21 @@ class Parser {
    * Adds releaseDate, vaultDate and estimatedVaultDate to all primes using
    * data from "Ducats or Plat".
    * @param {Item} item data to append vault data to
+   
    * @param {VaultData} vaultData to look up data for the {@param item}
+   * @param {module:warframe-items.Category} category of the data
+   * @param {WikiaData} wikiaData from wikia to apply 
    */
-  addVaultData(item, vaultData) {
+  addVaultData(item, vaultData, category, wikiaData) {
+    let vaultCategory = category;
+    if (item.type === 'Archwing') vaultCategory = 'archwings';
+    if (category === 'Upgrades') vaultCategory = 'mods';
+
+    if (!['weapons', 'warframes', 'mods', 'upgrades', 'archwings'].includes(vaultCategory.toLowerCase())) return;
```

Comment:
Sentinels also have primes
```suggestion
    if (!['weapons', 'warframes', 'mods', 'upgrades', 'archwings', 'sentinels'].includes(vaultCategory.toLowerCase())) return;
```

Reply:
Adding sentinels means adding a scraper for companions (https://warframe.fandom.com/wiki/Module:Companions/data). I can add this but I'm not sure if this would break anything else that would include companions.

Reply:
also looks like they also already have it https://api.warframestat.us/items/search/Carrier%20Prime/

Reply:
Well this pr is removing the data that sentinels already have, so maybe just use the ogg for it and not the wikia? or also add the  wikia data?

Reply:
Just diff the data that i provided and you'll see the problem
`jsondiffpatch ./itemsOld.json ./itemsNew.json`